wiki/People/Community_Contributors/fr.md

```diff
@@ -1,3 +1,7 @@
+---
+outdated_translation: true
```

Comment:
I noticed that the `outdated_since` tag has not been added, but this is not reported by the CI.